desktop/apps/home/routes.coffee

```diff
@@ -83,6 +83,7 @@ fetchMetaphysicsData = (req)->
 
       res.locals.sd.HERO_UNITS = heroUnits
       res.locals.sd.USER_HOME_PAGE = home_page.artwork_modules
+      res.locals.sd.HIGHLIGHT_HEADER_SEARCH = req.query?.highlight_header_search
```

Comment:
Yea, so this will only add this to `sd` on the home page, but I think the header update is more global than that right? Or is it just on the homepage?

For the more global effect, we can move this to a middleware.  [This locals one](https://github.com/artsy/force/blob/d1014370c317576e25d2f2e9b1d023bb7df87952/lib/middleware/locals.coffee#L33) seems to be the most appropriate.